### DISCUSSION TITLE
$("loop.currentItem") is always undefined

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6385] [Help] [answered]
I am trying to write an automation that will periodically update the rows.

I can query for the rows I need, but in the loop, I couldn't get the row data. `$("loop.currentItem")` is always undefined. 
If I try to print out `{{ loop.currentItem }}` in the handlebar it does have a value. Only not working in javascript.

Is this a bug?

![Screenshot 2022-06-20 at 2 56 43 PM](https://user-images.githubusercontent.com/1195573/174542852-8a16da74-8b6a-4d31-adde-8a4a28d238a5.jpg)

**Comment by melohagan** [comment, marked as answer]
Thanks for reporting @magickaito 

It is a bug! https://github.com/Budibase/budibase/issues/6387

I believe it does work if you use Handlebars in the meantime.

**Reply by magickaito** [reply]
ok!
